.golangci.yaml

```diff
@@ -49,17 +49,19 @@ linters-settings:
       - nilness
   goimports:
     local-prefixes: github.com/cilium/cilium
-  misspell:
-    # Correct spellings using locale preferences for US or UK.
-    # Default is to use a neutral variety of English.
-    # Setting locale to US will correct the British spelling of 'colour' to 'color'.
-    # locale: US
-    ignore-words:
-      - rela # This is for elf.SHT_RELA
```

Comment:
Was this change intentional?

Reply:
It was. iirc the code this was necessary for was removed with the alignchecker rewrite.

Reply:
Ideally yeah, in particular I asked the question because the commit message didn't mention it at all, so I wasn't sure how it related to the linting changes.

Reply:
Sure thing! :+1: